### PR TITLE
JEK-50: Embedded the feedback form into the page

### PR DESCRIPTION
**Pull request by dr3amf4ll**

### CODE REVIEW

File: gh-pages-stub/_includes/feedback.html [hunk 1]
```diff
@@ -1,49 +1,34 @@
...
+        <input type="text" class="form-control form__input" name="name" maxlength="40" data-minlength="3" placeholder="Enter name" data-error="Please enter your name" required="required" />
+      </div>
+
+      <div class="form-group col-md-6 col-sm-6 col-xs-12">
+        <input type="email" class="form-control form__input" name="email" data-minlength="3" maxlength="40" placeholder="Enter email" data-error="This email address is invalid" required="required" />
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label class="checkbox-inline form__radio"><input type="radio" name="type" autocomplete="off" checked="checked" value="Compliment" />Compliment</label>
```
Comment (Elena363) [inline gh-pages-stub/_includes/feedback.html]: @dr3amf4ll  Let's remove these radiobuttons? they look redundant

File: gh-pages-stub/_includes/feedback.html [hunk 2]
```diff
@@ -1,49 +1,34 @@
...
+      <textarea type="text" class="form-control form__textarea" name="description" data-minlength="5" rows="3" placeholder="Description (at least 5 symbols)" maxlength="320" required="required"></textarea>
+    </div>
+
+    <div class="form-group text-center">
+      <button type="submit" class="btn btn-primary form__submit">Submit</button>
+    </div>
+  </fieldset>
+</form>
+
 <script src="{{ site.baseurl }}/public/js/validator.js"></script>
```
Comment (Elena363) [inline gh-pages-stub/_includes/feedback.html:27]: I suggest to validate the values in inputs on keyup event as the validation on focus leave confuses. It should be fast to implement, shouldn't it? @dr3amf4ll

File: gh-pages-stub/public/css/styles.css [hunk 3]
```diff
@@ -1018,3 +1005,41 @@ table p {
 .small-width {
   max-width: 300px;
 }
+
+/* Form */
```
Comment (Elena363) [inline gh-pages-stub/public/css/styles.css:1008]: can u, please, remove the opacity for the disabled button & decrease the paddings above/beneath the form

Comment (Elena363) [reply gh-pages-stub/public/css/styles.css:1008]: @dr3amf4ll

Comment (Elena363) [reply gh-pages-stub/public/css/styles.css:1008]: & make text color white, grey text is difficult to distinguish on the orange bg.

File: gh-pages-stub/_includes/feedback.html (hunk 2 above)
Comment (dr3amf4ll) [reply gh-pages-stub/_includes/feedback.html:27]: Current validation library allows only such type of handling

File: gh-pages-stub/_includes/feedback.html (hunk 1 above)
Comment (dr3amf4ll) [reply gh-pages-stub/_includes/feedback.html]: Removed

File: gh-pages-stub/public/css/styles.css (hunk 3 above)
Comment (dr3amf4ll) [reply gh-pages-stub/public/css/styles.css:1008]: Done

**Pull request merged**